joomla/joomla-cms
# [4.0] Various improvements to the Local FileSystem plugin

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/20851

### Summary of Changes
1. Modify plugin description to inform of its usage
2. Modify Media Manager Alert when no filesystem plugin is enabled by adding a link to the plugins manager
3. Correct the hard coded "Local" name given to the plugin by allowing use of default lang string.
4. Alpha order lang strings and cs for the spaces before and after the `=` sign.

### Testing Instructions
Before patch
Edit the plugin, enable it, test name display in Media manager.
Disable plugin, test display in Media Manager.

After patch: test custom name and default language name setting 
Check description.
Disable plugin and see Alert in Media Manager.


### Result after patch

<img width="1278" alt="screen shot 2018-06-24 at 11 45 04" src="https://user-images.githubusercontent.com/869724/41817888-12de06a8-77a4-11e8-98dd-b8854af23214.png">


<img width="809" alt="screen shot 2018-06-24 at 11 43 27" src="https://user-images.githubusercontent.com/869724/41817875-ede8aa42-77a3-11e8-9204-d60399e102a1.png">

<img width="1212" alt="screen shot 2018-06-24 at 09 46 21" src="https://user-images.githubusercontent.com/869724/41820228-66795a54-77ce-11e8-80a8-55e290959fc4.png">
